package.json

```diff
@@ -42,5 +44,7 @@
   "devDependencies": {
     "concurrently": "^5.3.0",
     "json-server": "^0.16.3"
```

Comment:
I think this line needs a comma after it. And the line below should have a colon separating the name and version number

Reply:
Fixed!